File: src/main/java/es/alejandro12120/kqueue/kQueue.java
package es.alejandro12120.kqueue;

import es.alejandro12120.kqueue.servers.ServerManager;
import org.bukkit.plugin.java.JavaPlugin;

public class kQueue extends JavaPlugin {

    private ServerManager serverManager;

    @Override
    public void onEnable() {
        this.serverManager = new ServerManager();

        this.getLogger().info("kQueue has been enabled!");
    }


}

File: src/main/java/es/alejandro12120/kqueue/kQueue.kt
package es.alejandro12120.kqueue

import org.bukkit.plugin.java.JavaPlugin

class kQueue : JavaPlugin() {

    companion object {
        lateinit var instance: kQueue
            private set
    }

    override fun onEnable() {
        instance = this

        logger.info("kQueue has been enabled")
    }

}